packages/bubble/Bubble.tsx

```diff
@@ -0,0 +1,90 @@
+import { h, Component, prop } from 'skatejs';
+import styles from './Bubble.scss';
+import { css } from '../../utils/css';
+
+const BubbleTypes = {
+  top: 'top',
+  left: 'left',
+  right: 'right',
+  bottom: 'bottom',
+};
+
+type BubbleTypesType = typeof BubbleTypes;
+
+// public
+interface BubbleProps extends JSX.HTMLProps<HTMLElement | any> {
+  type?: keyof BubbleTypesType,
+  isDisplayed?: boolean,
+  disableAutoShowHide?: boolean
+}
+
+export class Bubble extends Component<BubbleProps> {
+  _props: BubbleProps;
+  static get is() { return 'bl-bubble' }
+  static get props() {
+    return {
+      type: prop.string({
+        attribute: true
+      }),
+      isDisplayed: prop.boolean({
+        attribute: true
+      }),
+      disableAutoShowHide: prop.boolean({
+        attribute: true
+      })
+    }
+  }
+
+  type = 'right';
+  isDisplayed = false;
+  disableAutoShowHide = false;
+
+  private handleMouseOver() {
```

Comment:
I think this might lead to an unpredictable behavior in some cases as - there are two sources:
- mouse
- focus

If we merge it it would might got displayed based on focus and then hidden based on moseover...